_15thnight/core.py

```diff
@@ -216,3 +216,26 @@ def send_help_message(user, message):
         recipients=[current_app.config.get('SUPPORT_EMAIL', '')]
     )
     queue_send_email.apply_async(kwargs=dict(message=message))
+
+
+def send_out_alert_closed(alert):
+    #TODO: Better date formatting
+    #TODO: Send out which needs were just resolved, to the outstanding providers
```

Comment:
I am going to say we should push ahead with it as-is. I think what is currently here is fine for now, and we need to deliver something ASAP.